src/lib/AlertWindow.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import ModalWindow from "./ModalWindow";
+import Button from "./Button";
+
+const InfoText = styled.div`
+  text-align: center;
+`;
+
+const ButtonRow = styled.div`
+  text-align: center;
+  margin-top: 32px;
+  
+  button {
+    margin: 0 16px;
+  }
+`;
+
+const AlertWindow = (props) => (
+  <ModalWindow {...props}>
+    <InfoText>{props.info}</InfoText>
+    <ButtonRow>
+      {props.buttons.map((button, idx) =>
+        <Button key={button.name + idx} rounded size="medium" onClick={button.onClick} theme={button.theme}>
```

Comment:
Would be great if you split every prop to a new line for components with 2+ props (unless they are short and concise in certain scenarios)

Reply:
Done